[Bulk] Validation: Fixed issues in ODatadocs (#219)
MicrosoftDocs/OData-docs: Odata-docs/connectedservice/choosing-what-to-generate.md

---
title: "Choosing what to generate"

> description: OData Connected Service allows you to select which types and methods to include in the generated code.

Reviewer: ```suggestion
description: Describes how to select which types and methods to include in the generated code.
```
